source/api/presentation/2.1/index.md

```diff
@@ -423,6 +423,15 @@ This pattern may be used in `label`, `description`, `attribution` and the `label
 
 Note that [RFC 5646][rfc5646] allows the script of the text to be included after a hyphen, such as `ar-latn`, and clients should be aware of this possibility. This allows for full internationalization of the user interface components described in the response, as the labels as well as values may be translated in this manner; examples are given below.
 
+In the case where multiple values are supplied, clients _MUST_ use the following algorithm to determine which values to display to the user.  
+
+* If none of the values have a language associated with them, the client _MUST_ display all of the values.
+* Else, the client should try to determine the user's language preferences.  If the user's language preferences are unknown, then the client _MUST_ supply a language preference. Then:
```

Comment:
It does seem a bit odd to say a client must _supply_.  It seems to me we are just saying that the client has to do something, perhaps make an arbitrary guess or, more likely, have a built in default of the language preference of the client author. I think the should in the Else clause should be RFC2119, and then perhaps change the following sentence to something weaker. Perhaps:

  *  Else, the client _SHOULD_ try to determine the user's language preferences in preference to using a default setting. Then: